### PR TITLE
[9.18] Added delete merged branches API function

### PR DESCRIPTION
**Pull request by fgoppelt**

### CODE REVIEW

File: lib/Gitlab/Api/Projects.php [hunk 1]
```diff
@@ -891,4 +891,13 @@ public function addProtectedBranch($project_id, array $params = [])
     {
         return $this->post($this->getProjectPath($project_id, 'protected_branches'), $params);
     }
+
+    /**
+     * @param $project_id
```
Comment (GrahamCampbell) [inline lib/Gitlab/Api/Projects.php]: invalid syntax

**Comment by GrahamCampbell** [comment]
Ping @fgoppelt. Do you plan on completing this PR please?

**Comment by fgoppelt** [comment]
Hey, sorry for the late response. I will complete this PR in a couple of days.

File: lib/Gitlab/Api/Projects.php (hunk 1 above)
Comment (fgoppelt) [reply lib/Gitlab/Api/Projects.php]: Added missing type hint

**Comment by GrahamCampbell** [comment]
Thanks. :)

**Pull request merged**